tests/Controller/OcrControllerTest.php

```diff
@@ -27,6 +27,7 @@ class OcrControllerTest extends OcrTestCase {
 	 * @covers OcrController::getLangs
 	 * @param string[] $getParams
 	 * @param string[] $expectedLangs
+	 * @return void
```

Comment:
Linting was failing for me.

Reply:
Ah right. Does that mean linting is failing on this in current main as well?

Reply:
yes it's failing on main as well.

`  1x: Method "Psr\Log\LoggerAwareInterface::setLogger()" might add "void" as a native return type declaration in the future. Do the same in implementation "MediaWiki\OAuthClient\Client" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in OcrControllerTest::testGetLang with data set #0 from App\Tests\Controller`

Reply:
Oh interesting, is that showing as an error for you? It's a deprecation warning for me, perhaps you have some other env var set to elevate it?

The warnings I see are:

```
Remaining indirect deprecation notices (4)

  1x: Method "Psr\Log\LoggerAwareInterface::setLogger()" might add "void" as a native return type declaration in the future. Do the same in implementation "MediaWiki\OAuthClient\Client" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in OcrControllerTest::testGetLang with data set #0 from App\Tests\Controller

  1x: The "Symfony\Bridge\Doctrine\Logger\DbalLogger" class implements "Doctrine\DBAL\Logging\SQLLogger" that is deprecated Use {@see \Doctrine\DBAL\Logging\Middleware} or implement {@see \Doctrine\DBAL\Driver\Middleware} instead.
    1x in OcrControllerTest::testGetLang with data set #0 from App\Tests\Controller

  1x: Method "Symfony\Component\Console\Command\Command::execute()" might add "int" as a native return type declaration in the future. Do the same in child class "Doctrine\Bundle\DoctrineBundle\Command\CreateDatabaseDoctrineCommand" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in OcrControllerTest::testGetLang with data set #0 from App\Tests\Controller

  1x: Method "Symfony\Component\Console\Command\Command::execute()" might add "int" as a native return type declaration in the future. Do the same in child class "Doctrine\Bundle\DoctrineBundle\Command\DropDatabaseDoctrineCommand" now to avoid errors or add an explicit @return annotation to suppress this message.
    1x in OcrControllerTest::testGetLang with data set #0 from App\Tests\Controller
```

Which I agree we should fix :-) but they should probably be in separate PRs.